lib/Module/Build/Base.pm

```diff
@@ -1079,8 +1079,10 @@ sub subclass {
   open(my $fh, '>', $filename) or die "Can't create $filename: $!";
   print $fh <<EOF;
 package $opts{class};
+use strict;
```

Comment:
This will break if the subclass code was not in fact strict.

Reply:
I'm happy to take that out. My goal is the our statements not the strict.

Reply:
Agreed, adding just the ours should be fine.

Reply:
done.